service/controller/key/key.go

```diff
@@ -474,16 +446,24 @@ func NetworkDNSBlock(servers []net.IP) string {
 	return dnsBlock
 }
 
-func NetworkNTPBlock(servers []net.IP) string {
-	var ntpBlockParts []string
-
-	for _, s := range servers {
-		ntpBlockParts = append(ntpBlockParts, fmt.Sprintf("NTP=%s", s.String()))
+// FindNodeCondition returns the condition of the given type from the node. The second return value indicates if the condition was found.
+func FindNodeCondition(node corev1.Node, conditionType corev1.NodeConditionType) (corev1.NodeCondition, bool) {
+	for _, condition := range node.Status.Conditions {
+		if condition.Type == corev1.NodeReady {
```

Comment:
`conditionType` is not being used in the fuction. I'm assuming you want to check for it here?

Reply:
Good catch. I'll fix it.